spikes/harbor/harbor.yml

```diff
@@ -0,0 +1,247 @@
+# Configuration file of Harbor
+
+# The IP address or hostname to access admin UI and registry service.
+# DO NOT use localhost or 127.0.0.1, because Harbor needs to be accessed by external clients.
+hostname: 0.0.0.0
+
+# http related config
+http:
+  # port for http, default is 80. If https enabled, this port will redirect to https port
+  port: 80
+
+# https related config
+https:
+  # https port for harbor, default is 443
+  port: 443
+  # The path of cert and key files for nginx
+  certificate: /workspaces/istio/harbor/harbor-ssl.crt
+  private_key: /workspaces/istio/harbor/harbor-ssl.key
+
+# # Uncomment following will enable tls communication between all harbor components
+# internal_tls:
+#   # set enabled to true means internal tls is enabled
+#   enabled: true
+#   # put your cert and key files on dir
+#   dir: /etc/harbor/tls/internal
+
+# Uncomment external_url if you want to enable external proxy
+# And when it enabled the hostname will no longer used
+# external_url: https://reg.mydomain.com:8433
+
+# The initial password of Harbor admin
+# It only works in first time to install harbor
+# Remember Change the admin password from UI after launching Harbor.
+harbor_admin_password: Harbor12345
```

Comment:
I wouldn't expose this, I think this is fine since it's a Spike but moving forward we need to configure a driver to read the keys from keyvault (ngsa style)

Reply:
Good point Joaquin. I will add, though it's a spike, we're still committing the changes to this public repo, and a cred scanner might flag it. If there's a way to specify in the documentation to set the password, that should be the go-to.

Reply:
Just a note, if we're not setting the password in helm-values, it will still have default password which is committed in [Harbor-helm repo](https://github.com/goharbor/harbor-helm/blob/master/values.yaml#L377)